src/it/scala/CirceDebeziumSpec.scala

```diff
@@ -0,0 +1,222 @@
+import org.http4s.circe.CirceEntityCodec._
+
+import io.circe._, literal._
+import io.circe.parser._
+
+import fs2.kafka._
+
+class CirceDebeziumSpec
+  extends AsyncIOSpec
+  with Matchers
+  with BeforeAndAfterAll
+  with ScalaCheckPropertyChecks
+  with ForAllTestContainer
+  with scalatest.IOChecker {
+
+  
+  case class Debezium(field1: Int, field2: Short, field3: Long, field4: Boolean, field5: String, field6: Float, field7: Double, field8: Option[Int])
+
+  implicit val dbDecoder = Decoder.forProduct8("field1", "field2", "field3", "field4", "field5", "field6", "field7", "field8")(Debezium.apply)
+
+  implicit val intKeyDeserializer = Deserializer.lift { bytes =>
+    IO.fromEither(decode[DebeziumKey[Int]](new String(bytes, "UTF8")))
+  }
+
+  implicit val dbvValueDeserializer = Deserializer.lift { bytes =>
+    IO.fromEither(decode[DebeziumValue[Debezium]](new String(bytes, "UTF8")))
+  }
+
+  implicit val cs    = IO.contextShift(scala.concurrent.ExecutionContext.global)
+  implicit val timer = IO.timer(scala.concurrent.ExecutionContext.global)
+
+  lazy val kafka = KafkaContainer().configure { c => {
+    c.withNetwork(Network.builder().build())
+    ()
+  }}
+
+  lazy val kafkaUrl = kafka.networkAliases(0) + ":9092"
+
+  val pgUser = "postgres"
+  val pgPW   = "postgres"
+  val pgPort = 5432
+
+  lazy val postgres = PostgreSQLContainer(
+    dockerImageNameOverride = "debezium/postgres:11",
+    username                = pgUser,
+    password                = pgPW
+  ).configure { c => {
+    c.withNetwork(kafka.network)
+    ()
+  }}
+
+  lazy val jdbcUrl = "jdbc:postgresql://" + postgres.containerIpAddress + ":" + postgres.mappedPort(pgPort) + "/"
+
+  val debeziumPort = 8083
+
+  lazy val debezium = GenericContainer(
+    DockerImage(Left("debezium/connect:1.2")),
+    List(debeziumPort),
+    Map(
+      "BOOTSTRAP_SERVERS"    -> kafkaUrl,
+      "CONFIG_STORAGE_TOPIC" -> "config.store",
+      "OFFSET_STORAGE_TOPIC" -> "offset.store"
+    )
+  ).configure { c =>
+    val _ = c.withNetwork(kafka.network)                  // Kafka Connect needs to be able to see Kafka
+  }
+
+  lazy val debeziumUri = Uri.unsafeFromString("http://" + debezium.containerIpAddress + ":" + debezium.mappedPort(debeziumPort))
+
+  lazy val container = MultipleContainers(postgres, kafka, debezium)
+
+  lazy val flyway = Flyway.configure().dataSource(jdbcUrl, pgUser, pgPW).load()
+
+  // A transactor that gets connections from java.sql.DriverManager and executes blocking operations
+  // on an our synchronous EC. See the chapter on connection handling for more info.
+  lazy val transactor = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver",   // driver classname
+    jdbcUrl,                   // connect URL (driver-specific)
+    pgUser,                // user
+    pgPW,                // password
+    Blocker.liftExecutionContext(ExecutionContexts.synchronous) // just for testing
```

Comment:
I've see ExecutionContexts.synchronous hang Circle CI. I tried to find a why to "shutdown" but didn't succeed. Ended up using the global EC to solve the hanging. Just keep it in mind if it hangs Circle.